packages/core/src/contracts/tcr/civilTCR.ts

```diff
@@ -590,6 +591,10 @@ export class CivilTCR extends BaseWrapper<CivilTCRContract> {
       didCollectAmount = await this.getRewardClaimed(appealChallengeID, user);
     }
 
+    if (isVoterWinner && !didUserCollect) {
+      voterReward = await this.voterReward(appealChallengeID, salt as BigNumber, user);
```

Comment:
this is weird... didn't I add this already?

Reply:
This is for appeal user challenge data!